### PR TITLE
Address flake8 warnings

### PR DESCRIPTION
**Pull request by jwodder**
This caused black to be run on the modified files, hence the many reformattings.

Also configured flake8 to be compatible with black.

### CODE REVIEW

**Comment by yarikoptic** [comment]
FWIW, I have tested -- merge conflicts with #134 are minor and could easily be handled, so no problem to merge this one before #134

File: dandi/_version.py [hunk 1]
```diff
@@ -190,7 +199,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
     # starting in git-1.8.3, tags are listed as "tag: foo-1.0" instead of
     # just "foo-1.0". If we see a "tag: " prefix, prefer those.
     TAG = "tag: "
-    tags = set([r[len(TAG):] for r in refs if r.startswith(TAG)])
+    tags = set([r[len(TAG) :] for r in refs if r.startswith(TAG)])
```
[the remark this answers is not in the document]
Comment (jwodder) [reply dandi/_version.py:202]: Black did that, and I configured flake8 to be OK with it.  Black's rationale is [here](https://github.com/psf/black/blob/master/docs/the_black_code_style.md#slices).

**Comment by jwodder** [comment]
> Overall looks good, but I worry about doing anything about `versioneer.py` and `_version.py`. We do not want to maintain their forks, and also some changes (adding a space before `:`) didn't look right to my PEP8 eye

I've undone the changes to those two files and configured flake8 and black to ignore them.

**Comment by yarikoptic** [comment]
setup.cfg is a popular one these days ;) please resolve conflicts and push. Let's get flake8'ness done with

**Comment by jwodder** [comment]
@yarikoptic Conflicts rebased away.

File: dandi/_version.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yarikoptic) [reply dandi/_version.py]: ok.  I will try to remember this black particularity.

**Comment by yarikoptic** [comment]
Great, thanks @jwodder !  Let's proceed!

**Pull request merged**